spec/integration/dry/configurable/setting_spec.rb

```diff
@@ -377,13 +377,13 @@
     end
 
     it 'can be finalized' do
-      klass.setting :db, 'sqlite'
```

Comment:
Why did you have to remove this line?

Reply:
Thought that it's not needed, since we have a :db already, but on the second though we're testing that config could be modified. Put it back.

Reply:
Oh, right, we're overwriting nested `db: %i[user password]` here. Replaced with `klass.setting :kafka, 'kafka://127.0.0.1:9092'`